antarest/study/repository.py

```diff
@@ -13,6 +14,7 @@ class StudyMetadataRepository:
     """
 
     def save(self, metadata: Study) -> Study:
+        metadata.updated = datetime.now()
```

Comment:
the field updated doesn't exist. also maybe it shouldn't be updated always.
add a argument (update_modification_date: bool = False) so that when saving a study we can decide if we want to update the modification date or not.
for instance, when adding a group or changing the owner, we shouldn't update the modification date